src/config/drive.rs

```diff
@@ -57,6 +60,11 @@ impl<'d> Drive<'d> {
     }
 
     /// Host level path for the guest drive.
+    pub fn host_path(&self) -> &Path {
```

Comment:
`path_on_host` is supposed to be host path so I'm not sure why we need another field/getter for the same thing. Perhaps `path_on_guest` (or just `path`) is needed here?